### ISSUE TITLE
Chrome doesnt record audio?

### ISSUE THREAD

**Issue by ulissesvaquero** [streamproc/MediaStreamRecorder#36]
How i can record audio in chrome if stereo?

**Comment by jineshpaloor** [comment]
Same issue here. Audio recording is happening in firefox, but not in chrome.

**Comment by kevinleeTCA** [comment]
yep, we got the same issue, actual the video demo does not work on firefox, when I click the recorded video link, it just corrupted.
